src/pages/Facility/services/utils/useCurrentService.ts

```
@@ -0,0 +1,49 @@
+    queryFn: query(healthcareServiceApi.retrieveHealthcareService, {
+      pathParams: { facilityId: facilityId, healthcareServiceId: serviceId },
+    }),
+    staleTime: 1000 * 60 * 30, // cache for 30 minutes
+  });
+
+  return { facilityId, serviceId, service };
+}
+
+export function useCurrentServiceSilently() {
+  try {
+    return useCurrentService();
```

Biome (2.1.2)

[error] This hook is being called conditionally, but all hooks must be called in the exact same order in every component render.

For React to preserve state between calls, hooks needs to be called unconditionally and always in the same order.
See https://reactjs.org/docs/hooks-rules.html#only-call-hooks-at-the-top-level

(lint/correctness/useHookAtTopLevel)